testsuite/MDAnalysisTests/analysis/test_hole2.py

```diff
@@ -307,10 +308,13 @@ def test_output_level(self, tmpdir, universe):
                                        output_level=100)
                 h.run(start=self.start,
                       stop=self.stop, random_seed=self.random_seed)
-            assert len(rec) == 3
-            assert 'needs to be < 3' in rec[0].message.args[0]
-            assert 'has no dt information' in rec[1].message.args[0]
-            assert 'has no dt information' in rec[2].message.args[0]
+
+            assert len(rec) == 5
+
```

Comment:
Any reason for the insertion of an extra empty line here?

Reply:
Personal preference, but I reverted it back.